### PR TITLE
Support M1

### PR DESCRIPTION
**Pull request by rajatvig**

### CODE REVIEW

File: k8s/k8s.bzl [hunk 1]
```diff
@@ -51,7 +51,7 @@ def k8s_repositories():
         http_archive,
         name = "com_github_yaml_pyyaml",
         build_file_content = _com_github_yaml_pyyaml_build_file,
-        sha256 = "e9df8412ddabc9c21b4437ee138875b95ebb32c25f07f962439e16005152e00e",
+        sha256 = "ba59d7e97eb131d8f8f52d19cb124bb67772f4c7f4d14cb2919deb885ef8c572",
```
Comment (fejta) [inline k8s/k8s.bzl:54]: Any insight into why/when the sha changed?

Comment (rajatvig) [reply k8s/k8s.bzl:54]: Seems like it has been fixed. Will revert. [This](https://twitter.com/tgummerer/status/1488493440103030787?s=20&t=DaupVstH5BiSJd0BgLSa_Q) has the info. There is chat in the [issue](https://github.com/bazelbuild/rules_k8s/issues/681) as well

**Comment by fejta** [comment]
I suspect we'll need to wait for various caches to become consistent again.

**Comment by rajatvig** [comment]
> I suspect we'll need to wait for various caches to become consistent again.

Very likely but unsure. To test, just downloaded the archive and it has the new SHA. Ran a clean bazel build with a local patch for it and that too used the new SHA.

**Comment by moriarty** [comment]
> > I suspect we'll need to wait for various caches to become consistent again.
> 
> Very likely but unsure. To test, just downloaded the archive and it has the new SHA. Ran a clean bazel build with a local patch for it and that too used the new SHA.

```bash
while true; do curl --silent -L https://github.com/yaml/pyyaml/archive/5.1.2.zip | shasum -a 256; sleep 2; done
```

Try that and see if you get consistent results. It seems to vary based on location and ISP... hopefully it trickles through all the various CDNs

**Comment by rajatvig** [comment]
@fejta From what it looks like, most caches have now got it fixed.

**Comment by fejta** [comment]
Can you drop the last two commits from this PR?

**Comment by rajatvig** [comment]
@fejta Dropped the two commits but I do not have rights to rebuild

**Comment by rajatvig** [comment]
@fejta Could you retrigger the tests?

**Comment by fejta** [comment]
Mostly superseded by https://github.com/bazelbuild/rules_k8s/pull/697

**Pull request closed without merging**